### DISCUSSION TITLE
how to use send_tokens for latest cosmos hub code

### DISCUSSION THREAD

**Discussion by NataliaAvila-PM** [fetchai/cosmpy#400] [Q&A]
HI everybody. First time a reach here.

I have been trying to use cosmpy with 0.50 cosmos hub code but facing some issues.
Any chance you can give me steps (or at least some hints) how to use the send_tokens function for latest cosmos hub code e.g. sdk 0.50. 

Thank you in advance.

**Comment by kshipra-fetch** [comment]
Hey @NataliaAvila-PM 

Thank you for reaching out! 

You can refer to the [Fetch.ai guide on sending tokens](https://fetch.ai/docs/guides/agents/intermediate/send-tokens) for detailed instructions and examples. 

Feel free to ask if you have any further questions!
